server/app/assets/stylesheets/styles.css

```diff
@@ -153,4 +153,37 @@
     cursor: default;
   }
 
+  .question-bank-add-button {
```

Comment:
Can you add a comment for the purpose of these classes are to help reduce the page load size cause by duplication of all these on each question. You can remove the line by line comments.